python/tests/integration/test_sse_cors_integration.py

```
@@ -5,67 +5,13 @@
 import httpx
 import pytest
 
+from .apps import app_module
+
 pytestmark = pytest.mark.server_integration
 
 
 def _make_sse_project(make_server_project):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `_make_sse_project`

(ANN202)